### ISSUE TITLE
[New Issue] Red Tape Runaround mission corrupted by 0.1.7

### ISSUE THREAD

**Issue by gir489returns** [Starfield-Community-Patch/Starfield-Community-Patch#1148]
### Platform

PC (Steam)

### DLCs

- [ ] Shattered Space

### Category

Missions & Dialogue

### Summary

When attempting the mission Red Tape Runaround, for some reason the quest becomes corrupted around the time you log into the HR woman's PC and attempt to delete the other applicant's records. For some reason, the globals are all set to 1 already, which is impossible because I had just received the quest. Setting at least one of the globals back to 0 allows the quest to proceed and complete the quest. However, during Red Tape Reclamation, while Hank is walking you to his ship to betray you, the game will crash. I confirmed nothing else was in my load order when loading the save other than Starfield Community Patch, and when it's just the SCP, the game crashes during his walk. When I remove the SCP, the game does not crash.

### Details

[Quicksave0_7EBBF132_526F62657274_090351_20240908210414_106_2_4.zip](https://github.com/user-attachments/files/16923710/Quicksave0_7EBBF132_526F62657274_090351_20240908210414_106_2_4.zip)

Here is a save about 10-15 seconds before I experience the crash.

### Game Version

1.13.61.0

### Language

English (EN)

### Have you recreated this issue on a new game?

Yes, I can recreate this issue.

### Reproduction Steps

Load save.
Wait 15 seconds.
Crash occurs.

**Comment by gir489returns** [comment]
With enough saving/unloading I was able to get passed this segment. I've also cleared a lot of really bad Creations out of my mod-list so they don't reload after I unloaded everything. They may have also been the problem.

**Comment by Deebz96** [comment]
I can see what led you to think that the SFCP was causing this, but I don't believe it's actually related at all. Furthermore, this quest itself doesn't seem to be related either. It also seems to be unrelated to your save file, as I was able to replicate it on my own character.

This seems to be a vanilla bug related to loading in this part of the Cydonia worldspace. For me, it was crashing when the area that his ship is parked at loads in, but only _some_ of the time. It did not happen every time I entered that area, and it was happening regardless of load order. Tried with my full usual load order, then with just the SFCP, then with no mods at all. Sometimes it worked fine, sometimes it crashed, with no rhyme or reason. If it didn't crash the first time, I could go back to my ship, then back to this area, and it would crash the second time.

Now, here's the weird part. I was not able to replicate this on a brand new, fresh character (started by typing coc cydoniaext01 from the main menu)... however, since I did that, I have not been able to replicate the crash at all anymore. I have restarted the game multiple times, try different load orders, tried both of our characters... can't get it to crash at all anymore. Very strange.

You might want to try replicating this with [Trainwreck](https://www.nexusmods.com/starfield/mods/5068), to get a crash log. That may help track down the source of the issue, but I make no promises with something this inconsistent.

I'll rename the title of this issue to better reflect the nature of the bug.

**Comment by gir489returns** [comment]
> This seems to be a vanilla bug related to loading in this part of the Cydonia worldspace. For me, it was crashing when the area that his ship is parked at loads in, but only _some_ of the time.

Yes! That's what lead me to think it was SFCP! I would disable everything but SFCP, load my save, it wouldn't crash, I'd restart with SFCP, then it would crash! Correlation ≠ causation though in this case.

But I'm glad someone else was able to replicate it, albeit at least for a short while.

> Now, here's the weird part. I was not able to replicate this on a brand new, fresh character (started by typing coc cydoniaext01 from the main menu)... however, since I did that, I have not been able to replicate the crash at all anymore. I have restarted the game multiple times, try different load orders, tried both of our characters... can't get it to crash at all anymore. Very strange.

I was able to get through the mission enough to get Hank back to the security sergeant, but because the sergeant is waiting on Hank to speak, I thought it locked up again, and ALT+F4d but in actually all I had to do was wait until Hank got close enough to the sergeant so he could speak although I did wait quite a long time. After restarting it crashed 2 more times before I was able to get him to do his "AH-HA! TIME TO BACKSTAB YOU!" nonsense.

**Comment by gir489returns** [comment]
This seems to also be happening on the UC Vigilant. All of my saves refuse to load and lock up from when I entered the UC Vigilant.

**Comment by Deebz96** [comment]
Well, I know I can’t replicate that one. Just completed the fleet questline very recently without a hitch. See if you can replicate it on a new character.

**Comment by gir489returns** [comment]
> Well, I know I can’t replicate that one. Just completed the fleet questline very recently without a hitch. See if you can replicate it on a new character.

It doesn't have a cell so I can't teleport it, and I don't feel like spending 5 hours to get out of the mines and go to it, so I'm just gonna have to skip that mission because I lost a lot of progress yet again to this stupid game crashing and freezing.

**Comment by Deebz96** [comment]
coc ssLC082Vigilance02

**Comment by gir489returns** [comment]
> coc ssLC082Vigilance02

Yeah I tried that but it gives me an error. Something about the ship being disabled. So I thought the cell doesn't exist.

**Comment by gir489returns** [comment]
I have given up trying to resuscitate the save file. It has become corrupted beyond saving, so I had to revert to a save from April, and I uninstalled a lot of unstable and broken Creations like Lyria and Robin Locke. I also uninstalled the Hyla II mansion. The only creations I have left are all the Bethesda Game Studios ones, Life of Crime, New Atlantis Waterfall Promenade Apartment, B320 pistol, Useful Mess Halls, SFCP (This), StarUI and the Cheat Terminal which I am developing myself.

I hope culling the creations on top of being more careful on what I install will make the later issue not repeat, although I'm going to also retest the two quests Red Tape Runaround and Red Tape Reclamation to see if they corrupt like they did last time. (Which I don't think they will)
